draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -336,7 +336,6 @@ obtain the public key using JWT VC Issuer Metadata as defined in (#jwt-vc-issuer
 - X.509 Certificates: If the recipient supports X.509 Certificates and the `iss` value contains an HTTPS URI, the recipient MUST
      1. obtain the public key from the end-entity certificate of the certificates from the `x5c` header parameter of the Issuer-signed JWT and validate the X.509 certificate chain accordingly, and
      2. ensure that the `iss` value matches a `uniformResourceIdentifier` SAN entry of the end-entity certificate or that the domain name in the `iss` value matches the `dNSName` SAN entry of the end-entity certificate.
-- DID Document Resolution: If a recipient supports DID Document Resolution and if the `iss` value contains a DID [@W3C.DID], the recipient MUST retrieve the public key from the DID Document resolved from the DID in the `iss` value. In this case, if the `kid` JWT header parameter is present, the `kid` MUST be a relative or absolute DID URL of the DID in the `iss` value, identifying the public key.
```

Comment:
This will limit the usability of the data model in some projects. @danielfett ?

May I propose an alternative text:

When `iss` is a DID, `jwk` JWT header MUST be present. The JWK MUST contain the required key members for a JWK of that key type and MUST contain the `kid` member and MAY contain other JWK members. The `kid` must have the value of the key identifier in the DID Document. The DID method MUST support key resolution in JWK(S) format following the scheme: {DID registry URL}/{iss}{kid}?format=jwk

Reply:
It does not limit anything.

Reply:
Then let's remove x509 :) It does not limit anything.